frontend/lib/resolution/call-graph.cpp

```diff
@@ -92,6 +92,27 @@ struct CalledFnCollector {
   }
   void exit(const AstNode* ast, RV& rv) {
   }
+
+  // TODO: How can we make this work through ``resolveFunction``, rather than
```

Comment:
Do you think we can add some special logic to `resolveFunction` that will resolve the parents and fetch the results? Or will that not work because the PoiScope will not line up?

Reply:
I don't think the PoiScope situation is clear enough at this time. I'd rather try and store this in the ``Context``, somehow (maybe by 'committing' the results once the parent is done resolving?).